### PR TITLE
docs(readme): Better type comparison

### PR DESCRIPTION
**Pull request by theKashey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,6 +267,26 @@ const element = <Component />
...
+Another way - compare "rendered" element type
+
+```js
+const element = <Component />
+console.log(element.type === (<Component/>).type) // true
+
+// better - precache rendered type
+const element = <Component />
+const ComponentType = (<Component />).type
+console.log(element.type === ComponentType // true
```
Comment (neoziro) [inline README.md]: Miss parenthesis

File: README.md [hunk 2]
```diff
@@ -267,6 +267,26 @@ const element = <Component />
...
 ```
 
+Another way - compare "rendered" element type
+
+```js
+const element = <Component />
+console.log(element.type === (<Component/>).type) // true
+
+// better - precache rendered type
+const element = <Component />
```
Comment (neoziro) [inline README.md]: Incompatible with PropTypes

**Comment by neoziro** [comment]
I think this is not a good thing to add in README, this is very confusing and it will generate a lot of issues.

File: README.md (hunk 2 above)
Comment (theKashey) [reply README.md]: :( Yeah, now I recall all the issues around it.

**Pull request merged**